.vscode/launch.json

```diff
@@ -7,7 +7,7 @@
             "type": "cppdbg",
             "request": "launch",
             "program": "${command:cmake.launchTargetPath}",
-            "args": [],
+            // "args": ["/mnt/fmi-library/build_cmake/Testing/BouncingBall3_me"], <-- fmi3_import_xml_test_me

```

Comment:
Some tests are parametrized on the same .c file, so it's not trivial to fix. I think it's better to just add the args here until we can fix it.